Switch Symbol Database to open beta (#23635)
DataDog/documentation: content/en/dynamic_instrumentation/expression-language.md

* Calling methods. Dynamic Instrumentation does not permit executing code that may have side effects. However, you can access `private` fields directly.
* Other native programming language syntax beyond what is described on this page.

> If you want to try out an improved user experience for Expression Language, consider opting into the [autocomplete and search open beta][6].

Reviewer: ```suggestion
Try our [autocomplete and search open beta][6] for an improved user experience of the Expression Language.
```

[3]: /metrics/types/?tab=histogram#metric-types
[4]: /tracing/trace_collection/custom_instrumentation/java/#adding-spans
[5]: /tracing/trace_collection/custom_instrumentation/java/#adding-tags

> [6]: /dynamic_instrumentation/symdb/

Author: I have to move the files for that, which will break the diff. Maybe I do it towards the end of the review or in a separate PR all together?
